config/owasp/suppressions.xml

```diff
@@ -21,8 +21,11 @@
     <cve>CVE-2022-45688</cve>
   </suppress>
   <suppress until="2023-08-31">
-    <cve>CVE-2023-35116</cve><!-- 2023-08-02 latest version is 2.15.2 and is still vulnerable -->
     <cve>CVE-2023-34034</cve>
     <cve>CVE-2020-8908</cve>
   </suppress>
```

Comment:
Can maybe merge the above and below suppressions as they do the same thing, just bump the august suppression to september